crates/noirc_frontend/src/graph/mod.rs

```diff
@@ -29,6 +29,12 @@ impl CrateId {
 #[derive(Debug, Clone, PartialEq, Eq, Hash, Ord, PartialOrd)]
 pub struct CrateName(SmolStr);
 
+impl CrateName {
+    pub fn is_valid_name(name: &str) -> bool {
```

Comment:
```suggestion
   fn is_valid_name(name: &str) -> bool {
```
Don't make it pub unless necessary.

Reply:
Ah, forgot to remove this in last commit.